### DISCUSSION TITLE
A new configuration env variable for custom URL prefix

### DISCUSSION THREAD

**Discussion by abdullah-retorio** [langfuse/langfuse#2747] [Ideas]
### Describe the feature or potential improvement

Hello there!

First of all, have a nice week.

We are self-hosting langfuse on our VM with also projects running. I am trying to redirect with Envoy.

Currently langfuse uses "localhost:3000/*" prefix default. We want to serve from "localhost:3000/langfuse/* . Could you please bring an environment variable that we can always add a custom prefix of langfuse routing?

**Comment by marcklingen** [comment]
Thanks for sharing this! We are tracking this request here: https://github.com/orgs/langfuse/discussions/2400

Open for contributions on this

**Reply by abdullah-retorio** [reply]
Thank you so much for your quick response! I checked open issues but I didn't find similar issues there. I will track that one!

**Reply by marcklingen** [reply]
no worries, happy to help

do you feel comfortable contributing this? have a look at the thread for more details

**Reply by abdullah-retorio** [reply]
I really would love to do this but currently, our company is working hard to bring new features.

ASAP our workload goes down, I would love to contribute. I have an idea of creating a self-hosting guide on Azure VMs with Envoy + TLS. :)

**Reply by marcklingen** [reply]
Sounds great, feel free to follow-up once you might get to this
